Detectors/TOF/simulation/src/Digitizer.cxx

```diff
@@ -110,9 +123,18 @@ int Digitizer::process(const std::vector<HitType>* hits, std::vector<Digit>* dig
   for (auto& hit : *hits) {
     // TODO: put readout window counting/selection
     //  neglect very slow particles (low energy neutrons)
-    if (hit.GetTime() > 1000) { // 1 mus
+    if (hit.GetTime() > MAX_HIT_TIME) { // 1 mus
       continue;
     }
+
+    // discard hits arriving before the minimum readout window
+    auto hit_ro_window = getReadoutWindow(double(hit.GetTime()) + mEventTime.getTimeNS() /*+ Geo::LATENCYWINDOW*/);
+
+    // discard hits arriving too early
+    if (hit_ro_window < mReadoutWindowMin) {
```

Comment:
As you prefer (it may be better to mark it as TFstart or something like that instead of Min), in any case I would change that line to avoid to digitize before of WindowCurrent.